langchain/chains/loading.py

```diff
@@ -200,19 +200,19 @@ def _load_llm_checker_chain(config: dict, **kwargs: Any) -> LLMCheckerChain:
 
 
 def _load_llm_math_chain(config: dict, **kwargs: Any) -> LLMMathChain:
-    if "llm" in config:
-        llm_config = config.pop("llm")
-        llm = load_llm_from_config(llm_config)
+    if "llm_chain" in config:
+        llm_chain_config = config.pop("llm_chain")
+        llm_chain = load_chain_from_config(llm_chain_config)
     elif "llm_path" in config:
```

Comment:
same comment here, should be llm_chain_path

Reply:
done